src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -60,31 +60,31 @@ export function Inquiries(req: Request, res: Response, next: NextFunction) {
   if (info.confirmed_buyer_at) {
     markCollectorAsLoyaltyApplicant(CURRENT_USER.accessToken)
       .then(profile => {
-        return res.redirect(req.baseUrl + "/thank-you")
+        res.redirect(req.baseUrl + "/thank-you")
       })
       .catch(err => console.error(err))
```

Comment:
> If you pass anything to the next() function (except the string 'route'), Express regards the current request as being in error and will skip any remaining non-error handling routing and middleware functions.

So looks like you should replace it with `.catch(next)`.

> If you want to handle that error in some way, you’ll have to create an error-handling route as described in the next section.

I assume Force has an error handler, and so doing it this way will make the error be shown however Force determines it should, but we may need a simple one in the dev server of reaction-force. Can you test if express has a default built-in error handler that’s helpful during dev?